model/godleyTableWindow.cc

```diff
@@ -487,10 +492,10 @@ namespace minsky
       }
     else if (r>0 && selectedCol==0)
       {
-        if (r!=selectedRow)
+        if (r!=selectedRow && !godleyIcon->table.initialConditionRow(selectedRow) && !godleyIcon->table.initialConditionRow(r))  // Cannot move Intitial Conditions row. For ticket 1064
           godleyIcon->table.moveRow(selectedRow,r-selectedRow);
       }
-    else if ((c!=selectedCol || r!=selectedRow) && c>0 && r>0)
+    else if ((c!=selectedCol || r!=selectedRow) && c>0 && r>0 && !godleyIcon->table.initialConditionRow(selectedRow) && !godleyIcon->table.initialConditionRow(r)) // Cannot swap individual cells in IC row. For ticket 1064
```

Comment:
This is probably a little too restrictive. Just check that (c,r)!=(0,1) and (selectedCol,selectedRow)!=(0,1)

Reply:
Okay, cells other than (1,0) in the first row can now also be swapped out for a (selectedCol!=0 || selectedRow!=1) condition.